src/basic_potentials.jl

```diff
@@ -119,29 +119,28 @@ function pairwise_electrostatic_acceleration!(dv,
     p::ElectrostaticParameters,
     pbc::BoundaryConditions)
 
-    force = @SVector [0.0, 0.0, 0.0];
+    force = @SVector [0.0, 0.0, 0.0]
     ri = @SVector [rs[1, i], rs[2, i], rs[3, i]]
     @inbounds for j = 1:n
         if !in(j, exclude[i])
```

Comment:
`in` on a vector is kind of expensive vs a set. Could the data structure change here?

Reply:
This one has me stumped, actually. I went ahead and transitioned to a Dict of Sets, thinking it a no brainer, but it's actually slower by quite a margin (30-40%) specifically in that membership check. I have no idea why, to be honest.